### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

File: src/NFTAdapter.sol [hunk 1]
```diff
@@ -49,27 +49,24 @@ contract NFTAdapter is DSNote {
...
+    function join(address urn, uint256 obj) external note {
         require(uint256(uint160(obj)) == obj, "obj-overflow");
 
         gem.transferFrom(msg.sender, address(this), obj);
-        vat.slip(ilkName(kin, obj), urn,  ONE);
+        vat.slip(ilkName(kin, obj), urn,  1);
     }
 
-    function exit(bytes32 urn, address guy, uint256 obj) external note {
+    function exit(address guy, uint256 obj) external note {
```
Comment (asymmetric) [inline src/NFTAdapter.sol]: I think we use `usr` now in dss, so should we use it here too?

**Review by asymmetric — COMMENTED** [review]
Just one nitpick, otherwise LGTM.

**Review by asymmetric — APPROVED** [review]

File: src/NFTAdapter.sol (hunk 1 above)
Comment (gbalabasquer) [reply src/NFTAdapter.sol]: Oh yes, it is using `usr` in the other ones. Let's update it.

Comment (gbalabasquer) [reply src/NFTAdapter.sol]: Done!

**Pull request merged**